File: SchoolSync_SpringBoot/backend/src/main/java/com/schoolsync/dto/TeacherDTO.java
package com.schoolsync.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class TeacherDTO {
    private Long teacherId;
    private String username;
    private String subject;
    private String email;
    private String phone;
}

File: Backend in .NET/DTOs/TeacherDto.cs
namespace SmartSchoolAPI.DTOs;

public class TeacherDto
{
    public int? TeacherId { get; set; }
    public string Username { get; set; } = string.Empty;
    public string Password { get; set; } = string.Empty;
    public string? Email { get; set; }
    public string? Phone { get; set; }
    public string Subject { get; set; } = string.Empty;
}

